Sprint-3/quote-generator/index.html

```diff
@@ -11,5 +11,12 @@ <h1>hello there</h1>
     <p id="quote"></p>
     <p id="author"></p>
     <button type="button" id="new-quote">New quote</button>
+    <div>
```

Comment:
You need to link your css here! because of that you page looks like this: 
![image](https://github.com/user-attachments/assets/1d915f99-c837-4a9f-ae1f-aac65320d98d)